problems/arrayMethods.js

```diff
@@ -91,15 +151,29 @@ const targetCount = () => {};
  * @returns {Object} A mapping of elements to their occurrence count.
  */
 
-const counterObject = () => {};
+const counterObject = (items) => {
+  let counter = {};
+  items.forEach((el) => {
+    if (counter[el] !== undefined) {
```

Comment:
can be condensed to counter[el] because if it has a value we know it's not undefined.